mne/epochs.py

```diff
@@ -2247,6 +2352,25 @@ def _read_one_epoch_file(f, tree, preload):
 
         events, mappings = _read_events_fif(fid, tree)
 
+        #   Metadata
+        metadata = None
+        metadata_tree = dir_tree_find(tree, FIFF.FIFFB_MNE_METADATA)
+        for dd in metadata_tree[0]['directory']:
+            kind = dd.kind
+            pos = dd.pos
+            if kind == FIFF.FIFF_DESCRIPTION:
+                tag = read_tag(fid, pos)
+                if tag.data == 'None':
+                    continue
+                if _check_pandas_installed(strict=False) is False:
+                    warn('Metadata found but pandas is not installed, not '
+                         'loading.')
+                    continue
+                import pandas as pd
```

Comment:
This should have `try/except` and raise an informative error that pandas must be installed to read an Epochs instance that contains metadata